test/test_index_string.cpp

```diff
@@ -1688,14 +1688,40 @@ TEST(StringIndex_Fuzzy)
                 }
             }
 
+
+            for (size_t r = 0; r < 5 + 1000 * TEST_DURATION; r++) {
+                size_t chunks;
+                if (fastrand() % 2 == 0)
+                    chunks = fastrand() % 4;
+                else
+                    chunks = 2;
```

Comment:
yes, it must be based towards 2 because common suffixes and common prefixes are important to test in index